go.mod

```diff
@@ -1,14 +1,13 @@
-module github.com/crossplanebook/provider-template
+module github.com/crossplane/provider-template
 
 go 1.13
 
 require (
-	github.com/crossplane/crossplane-runtime v0.9.1-0.20200923111851-e1fc0493b2e8
-	github.com/crossplane/crossplane-tools v0.0.0-20200923030414-95b434323cd4
-	github.com/go-logr/zapr v0.1.1 // indirect
-	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
+	github.com/crossplane/crossplane-runtime v0.9.1-0.20201001234308-3f1afd106a8c
+	github.com/crossplane/crossplane-tools v0.0.0-20201001224552-fb258cc0eb30
+	github.com/crossplanebook/provider-template v0.0.0-20200617152219-f62849cb7889 // indirect
```

Comment:
I think this can be cleaned out, not sure if an import somewhere is causing it to stick around 🤔

Reply:
Ah damn - yeah I cleared it out at least once. Not sure where it's coming from either.